src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1578,11 +1594,13 @@ internal void DisconnectOutput(int portData, int inPortData, NodeModel nodeModel
         private void ClearTooltipText()
         {
             ToolTipText = "";
+            infos.RemoveWhere(x => x.State == ElementState.Warning || x.State == ElementState.Error);
```

Comment:
so now the clearing the tooltip clears all the warnings/errors ?
With a little refactor I think we can get rid of the Tooltip from the Model and move it/replace with the new InfoBubble system

Reply:
@aparajit-pratap @pinzart90 This PR is getting bigger and bigger. Any chance we can merge and do follow up PR or they better be done together? I am asking since our team have some dependency on this

Reply:
@QilongTang ok, I'll merge this as-is and make a follow-up PR.

Reply:
@aparajit-pratap @pinzart90 Appreciate it! Sorry for the noise, just would like to unblock some work on our end so we can work in parallel on them. Cheers.